### ISSUE TITLE
Question: Gunicorn logging config conflicts with module logging

### ISSUE THREAD

**Issue by naxu-hiretual** [benoitc/gunicorn#2310]
I am encountering the following issue and wonder if anyone can help here:

When I start Gunicorn, I passed the logging config exactly as the example in this repo -
https://github.com/benoitc/gunicorn/blob/master/examples/logging.conf

In my module code, I have a own get_logger function, looks like this -

def get_logger(arg='root'):
    logger = logging.getLogger(arg)
    today = str(datetime.date.today())
    FORMAT = "%(levelname)s - %(asctime)s [%(filename)s || Line:%(lineno)s]: %(message)s"
    logging.basicConfig(filename='{}/{}.log'.format(LOG_DIR, today),
                        filemode='a', format=FORMAT, datefmt='%Y-%m-%d %H:%M:%S')
    logger.setLevel(logging.INFO)
    return logger

Even though I set the logging level to INFO in get_logger, it seems that the INFO messages are not written to the log file, while the WARNING messages got written to the warning.log in the gunicorn log directory. I figured that the root logger was reset by the gunicorn, so the setting in my self-defined function does not work here. I would like to solve the conflict and let gunicorn not set the root logger. 
Any help is appreciated!

**Comment by benoitc** [comment]
well yes. Gunicorn is using its own logger so until you're using it it will not be logged to the gunicorn error log.

**Comment by naxu-hiretual** [comment]
> well yes. Gunicorn is using its own logger `gunicorn.error` so until you're using it it will not be logged to the gunicorn error log.
Thanks for your comment, but I would like to clarify what I meant:
I am using the gunicorn logger, as I passed the logging conf file to gunicorn, and it uses three loggers: 
[loggers]
keys=root, gunicorn.error, gunicorn.access
The issue is that with gunicorn logger, I can not use the logger I defined myself for my own code (the get_logger function in my previous post).  
Hope it is clear and any suggestion is appreciated here. Thanks!
